azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/utils/WebAppUtils.java

```diff
@@ -18,17 +18,31 @@
 import com.microsoft.azure.management.resources.fluentcore.arm.Region;
 import com.microsoft.azure.maven.utils.AppServiceUtils;
 import com.microsoft.azure.maven.webapp.configuration.DockerImageType;
+import org.apache.commons.io.IOUtils;
 import org.apache.maven.plugin.MojoExecutionException;
 import org.apache.maven.plugin.logging.Log;
 import org.codehaus.plexus.util.StringUtils;
 
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
 public class WebAppUtils {
     public static final String SERVICE_PLAN_NOT_APPLICABLE = "The App Service Plan '%s' is not a %s Plan";
     public static final String CREATE_SERVICE_PLAN = "Creating App Service Plan '%s'...";
     public static final String SERVICE_PLAN_EXIST = "Found existing App Service Plan '%s' in Resource Group '%s'.";
     public static final String SERVICE_PLAN_CREATED = "Successfully created App Service Plan.";
     private static final String CONFIGURATION_NOT_APPLICABLE =
         "The configuration is not applicable for the target Web App (%s). Please correct it in pom.xml.";
+    public static final String JAR_CMD = ":JAR_COMMAND:";
+    public static final String FILENAME = ":FILENAME:";
+    public static final String DEFAULT_JAR_COMMAND = "-Djava.net.preferIPv4Stack=true " +
```

Comment:
why DEFAULT_JAR_COMMAND  is public?

Reply:
Fixed it, only set warning messages to public